Add validation to BestBid Phase

Scandie opened the pull request:
- BestBid Phase bid shouldn't be less or equal to SealedBid Phase max bid.

Can be merged now.

Scandie committed:
Add validation to BestBid Phase

- BestBid Phase bid shouldn't be less or equal to SealedBid Phase max bid.

yarsanich commented on openprocurement/auction/insider/utils.py:
```diff
@@ -171,6 +171,13 @@ def get_dutch_winner(auction_document):
         return {}
 
 
+def get_sealed_bid_winner(auction_document):
```
Додай `docstring` на цю функцію.

yarsanich commented on openprocurement/auction/insider/forms.py:
```diff
@@ -45,13 +47,21 @@ def validate_bid_value(form, field):
             raise e
     elif phase == BESTBID:
         # TODO: one percent step validation
```
Це TODO можна прибрати

yarsanich commented on openprocurement/auction/insider/forms.py:
```diff
@@ -45,13 +47,21 @@ def validate_bid_value(form, field):
             message = u'Bid value can\'t be less or equal current amount'
             raise ValidationError(message)
+        sealed_bid_winner = get_sealed_bid_winner(form.document)
+        sealed_bid_amount = sealed_bid_winner.get('amount')
```
Можеш зразу ж взяти `amount` без оголошення `sealed_bid_winner`.

Scandie committed:
Add test for get_sealed_bid_winner util function

- Fix test_server.py module due to new bestbid validation
- Add docstring for get_sealed_bid_winner util function

Scandie committed:
Update validation message
